### ISSUE TITLE
Incompatible with sphinxcontrib-video 0.2.1

### ISSUE THREAD

**Issue by adamtheturtle** [sphinx-contrib/confluencebuilder#977]
With the new `sphinxcontrib-video==0.2.1`, the `.. video::` directive gives:

```
    video_key, _, _ = self.assets.add(source_path, self.docname)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/adam/.virtualenvs/sphc/lib/python3.11/site-packages/sphinxcontrib/confluencebuilder/assets.py", line 87, in add
    return self._handle_entry(abs_path, docname, standalone=True)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/adam/.virtualenvs/sphc/lib/python3.11/site-packages/sphinxcontrib/confluencebuilder/assets.py", line 276, in _handle_entry
    hash_ = ConfluenceUtil.hash_asset(path)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/adam/.virtualenvs/sphc/lib/python3.11/site-packages/sphinxcontrib/confluencebuilder/util.py", line 60, in hash_asset
    with asset.open('rb') as file:
         ^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'open'
```

`sphinxcontrib-confluencebuilder==2.5.2`

**Comment by jdknight** [comment]
Thanks for the report. A change (#978) has been merged into the main branch and should be available next release (v2.6).

**Comment by jdknight** [comment]
v2.6 is now available on PyPI -- marking as closed.
